settings.php

```diff
@@ -102,7 +102,7 @@
 
         $settings->add(new admin_setting_configduration('tool_objectfs/expirationtime',
             new lang_string('settings:presignedurl:expirationtime', 'tool_objectfs'),
-            new lang_string('settings:presignedurl:expirationtime_help', 'tool_objectfs'), '', 10 * MINSECS));
+            new lang_string('settings:presignedurl:expirationtime_help', 'tool_objectfs'), 10, MINSECS));
```

Comment:
I think it should be `10 * MINSECS` if we want `10 mins`

Reply:
This should be minutes, and also 10 is waaaaayyy to short. This needs to be around the same as the session expiry.

$CFG->sessiontimeout now defaults to 8 hours, so lets make this expiry at least 2 hours